src/genie/libs/parser/iosxe/show_vlan_brief.py

```diff
@@ -0,0 +1,98 @@
+"""
```

Comment:
we already have show_vlan for iosxe please add the parser to that file instead of creating a new one